docker-files/zeeguu-api-core/Dockerfile

```diff
@@ -24,17 +25,39 @@ RUN python setup.py install
 # https://github.com/zeeguu-ecosystem/Python-Wordstats/blob/master/wordstats/config.py#L1
 # under /opt/Zeeguu-API/wordinfo.db
 COPY --chown=www-data:www-data Zeeguu-API /opt/Zeeguu-API
+RUN rm -rf /opt/Zeeguu-API/.git
 WORKDIR /opt/Zeeguu-API
 RUN pip install -r requirements.txt
-RUN python setup.py install
```

Comment:
Did you intend to remove this line?

Reply:
very good question. although the code runs anyway, it's still probably better to install the package. reinstated the line. although i'm not sure why actually!